app/helper_profile.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta name="description" content="SUMO real time support app for FirefoxOS" />
+  <meta name="viewport" content="width=device-width" />
+
+  <title>Welcome to BuddyUP</title>
+
+  <link rel="stylesheet" type="text/css" href="css/building_blocks/layout.css" media="screen" />
+  <link rel="stylesheet" type="text/css" href="css/building_blocks/headers.css" media="screen" />
+  <link rel="stylesheet" type="text/css" href="css/building_blocks/lists.css" media="screen" />
+  <link rel="stylesheet" type="text/css" href="css/building_blocks/input_areas.css" media="screen" />
+
+  <link rel="stylesheet" type="text/css" href="css/building_blocks/switches.css" media="screen" />
+  <link rel="stylesheet" type="text/css" href="css/building_blocks/value_selector.css" media="screen" />
```

Comment:
Have you checked all those building blocks are necessary? I'm thinking input_areas, switches and value_selector are not.